src/WebJobs.Script.WebHost/App_Data/secrets/HttpTrigger.json

```diff
@@ -1,3 +0,0 @@
-﻿{
```

Comment:
I'd like to leave all these example values in here as a sample to show people. I reference it in the wiki.